src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -175,12 +178,13 @@
             label="Delivery Address same as Mailing Address"
             hide-details
             v-model="inheritMailingAddress"
+            :disabled="disableSameDeliveryAddress"
             @change="setDeliveryAddressToMailingAddress()"
           />
         </v-col>
       </v-row>
 
-      <template v-if="!inheritMailingAddress">
+      <template v-if="!inheritMailingAddress || disableSameDeliveryAddress">
```

Comment:
Won't this affect other change filings?

I mean, is the rule behind this the same for all change filings (ie, BC + CA) ?

Reply:
Commented on all these but git didn't seem to apply them 😩 

Alterations are unable to edit addresses (this code block)
Corrections uses another sub template from below, since the design layout there is quite different.